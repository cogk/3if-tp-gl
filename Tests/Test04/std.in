3
oui
ça marche ?
µ£%µ%µ&é"'(-è_çà)=
