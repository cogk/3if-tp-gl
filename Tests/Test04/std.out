* Bienvenue dans AirWatcher 
| Utilisateur: (anonyme)       
* Menu Principal 

   1. Contribution…
   2. Analyse…
   3. Sources de données…
   4. Administration…
   5. Super Admin…
   6. Quitter AirWatcher

Entrez ici votre sélection : 
* Menu - Sources de données 

   1. Liste des sources
   2. Qualité des données
   3. Données médiocres
   4. Retour au menu principal

Entrez ici votre sélection : 
Entrée invalide, veuillez entrer un entier entre 1 et 4.
Vous n'avez pas choisi de menu.
* Menu Principal 

   1. Contribution…
   2. Analyse…
   3. Sources de données…
   4. Administration…
   5. Super Admin…
   6. Quitter AirWatcher

Entrez ici votre sélection : 
Entrée invalide, veuillez entrer un entier entre 1 et 6.
Vous n'avez pas choisi de menu.
